invenio_communities/assets/semantic-ui/js/invenio_communities/members/components/bulk_actions/SelectedMembers.js

```diff
@@ -28,13 +28,25 @@ export class SelectedMembers extends Component {
         <Header as="h4" className="ml-20">
```

Comment:
```suggestion
        <Header as="h3" className="ml-20">
```

From what I understand this heading is following the main heading in the modal ("Change roles", which should be h2)? If you want to change the size, use sizing classes, see https://react.semantic-ui.com/elements/header/#types-content